src/app/private/course/[courseId]/page.tsx

```diff
@@ -1,33 +1,34 @@
 import Header from "@components/Header";
```

Comment:
nit: For this page and the page.tsx for TAs, in cases where the content on the page doesn't fill up the whole page (e.g. if there are only a few TAs so the list is short) it still seems to be able to be scrolled, maybe we can only allow it to scroll when the content fills up the page